bricklayer/api/__init__.py

```diff
@@ -151,36 +161,44 @@ def mkdir(self, dir_path):
             )
         )
 
-    def backup_notebook(self, source_path, target_path, tmp_dir):
+    def backup_notebook(self, source_path, target_path, fmt:"DBC"):
         "Backup a notebook to another place in the workspace"
-        tmp_name = f'backup_{random.randint(0,1000)}'
+        tmp_dir = '/dbfs/tmp/'
+        tmp_name = 'backup'
         intermediate_location = pathlib.Path(tmp_dir).joinpath(tmp_name)
-        self.export_notebook(source_path, intermediate_location.as_posix())
+        print(intermediate_location.as_posix())
```

Comment:
We shall rather add a logger for this or just remove the print.

Reply:
```suggestion
```